src/index.css

```diff
@@ -1,7 +1,78 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Mulish:wght@400;500;600;700&display=swap');
+
+:root {
+  /* FONTS */
+
+  --Raleway: 'Raleway', sans-serif;
+  --Mulish: 'Mulish', sans-serif;
+
+  /* SIZES  */
+
+  --fs-12: 1.2rem;
+  --fs-14: 1.4rem;
+  --fs-16: 1.6rem;
+  --fs-20: 2rem;
+  --fs-24: 2.4rem;
+  --fs-26: 2.6rem;
+  --fs-40: 4rem;
+
+  --fw-normal: 400;
+  --fw-medium: 500;
+  --fw-semibold: 600;
+  --fw-bold: 700;
+
+  --main-container: 114rem;
+
+  /* COLORS */
+
+  --white-color: #ffffff;
+  --wetasphalt: #989cac;
+  --black: #000000;
+  --ruinedSmores: #020204;
+  --blackIsBlack: #0f1118;
+  --watermelon: #ff455b;
+
+  --gradient: linear-gradient(90deg, #ff455b 0%, #ff7245 100%);
+  --bottom-line: 2px solid;
+  --bottom-line-color: rgba(#ffffff, 0.05);
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
+
+html {
+  font-size: 62.5%; /* 10px */
```

Comment:
for html please use px :) only place where we can do that

Reply:
are you sure that 10 px is correct? usually it is about 16px (basic font size in mockups)